### DISCUSSION TITLE
How to change the drop marker color in CSS ?

### DISCUSSION THREAD

**Discussion by haoadoreorange** [piroor/treestyletab#3534] [Q&A] [answered]
As the title, I can't find how to change the drop marker (the line under of above the tab drop destination) even after inspecting :(

Please help :(

**Comment by piroor** [comment, marked as answer]
There is the hint:
https://github.com/piroor/treestyletab/blob/7b9e52ebcf7b9b91c4c26c2c149e3ae93aa12093/webextensions/sidebar/styles/drag-and-drop.css#L10

For example:
```css
:root {
  --tab-dropmarker: red;
}
```

**Reply by haoadoreorange** [reply]
omg thank you so much, I'm an idiot.

**Reply by haoadoreorange** [reply]
but just out of curiosity what the actual element that was doing it ? I couldn't find it for some reason, no changes I made to outline or border worked at all.
